lib/File/Find.pm

```diff
@@ -40,12 +40,14 @@ sub find (:$dir!, :$name, :$type, :$exclude = False, Bool :$recursive = True,
         next if $elem ~~ $exclude;
         take $elem if checkrules($elem, { :$name, :$type, :$exclude });
         if $recursive {
-            if $elem.IO ~~ :d {
-                @targets.append: dir($elem);
-                CATCH { when X::IO::Dir {
-                    $_.throw unless $keep-going;
-                    next;
-                }}
+            unless !$follow-symlinks and $elem.IO ~~ :l {
+            	if $elem.IO ~~ :d {
```

Comment:
Hey, any reason why you changed the original indentation here? Also, it may be worth considering (or noting in the documentation) whether the symlinks themselves should get “returned” if they're not followed.